l10n_br_simple/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2009  Renato Lima - Akretion
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Brazilian - Simple Accounting",
+    "summary": "Brazilian Simple Chart of Account",
+    "category": "Localization",
+    "license": "AGPL-3",
+    "author": "Akretion, " "Odoo Community Association (OCA)",
+    "website": "http://github.com/OCA/l10n-brazil",
+    "depends": ["account"],
```

Comment:
@renatonlima could you add "version": "12.0.1.0.0" in the manifest? the merge is failing because it is missing.